server/src/search.rs

```diff
@@ -634,3 +608,95 @@ fn highlight_matches(s: &String, search_tokens: &Vec<SearchToken>) -> String {
 
     s_highlighted.to_string()
 }
+
+// Parse the search against some known room formats and improve the
+// results display in this case. Room formats are hardcoded for now.
+fn parse_room_formats(
+    search_tokens: &Vec<SearchToken>,
+    hit: &MSHit,
+) -> Option<String> {
+    // Some building specific roomcode formats are determined by their building prefix
+    if search_tokens.len() == 2
+        && match search_tokens[0].s.as_str() {
+            "mi" => hit.id.starts_with("560") || hit.id.starts_with("561"),
+            "mw" => hit.id.starts_with("550") || hit.id.starts_with("551"),
+            "ph" => hit.id.starts_with("5101"),
+            "ch" => hit.id.starts_with("540"),
+            _ => false,
+        }
+        && !search_tokens[1].s.contains("@")
```

Comment:
Just some ideas:
- We could use the GitHub Wiki, or
- we could create an about page with explanations of room code formats

Reply:
I thought more about in the API-Documentation, to indicate to client developers, what characters should maybe get special highlighting when entering search-text, but on the website seems fine too.
I dont think that we need a github wiki, since we realistically dont have enough content to support that.

Reply:
This is definitly important once special query terms (such as filters) are implemented. Though the "@" here refers to the common Roomfinder format (`archname@building_id`) if you mean that.

Reply:
ah thanks for the clarification